src/m365/teams/commands/user/user-app-add.spec.ts

```diff
@@ -110,6 +115,96 @@ describe(commands.USER_APP_ADD, () => {
     } as any);
   });
 
+  it('adds app by name from the catalog for the specified user', async () => {
```

Comment:
Let's add test cases to verify that validation is working as expected for cases like:

- id and name both not specified
- id and name both specified